hrm-domain/hrm-service/migrations/20230926214506-create-client-profiles-tables.js

```diff
@@ -0,0 +1,317 @@
+/**
+ * Copyright (C) 2021-2023 Technology Matters
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see https://www.gnu.org/licenses/.
+ */
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  up: async queryInterface => {
+    // Profiles
+    await queryInterface.sequelize.query(`
+      CREATE SEQUENCE IF NOT EXISTS public."Profiles_id_seq"
+        INCREMENT 1
+        START 1
+        MINVALUE 1
+        MAXVALUE 9223372036854775807
+        CACHE 1;
+    `);
+    console.log('Sequence "Profiles_id_seq" created');
+    await queryInterface.sequelize.query(`
+    ALTER SEQUENCE public."Profiles_id_seq" OWNER TO hrm
+    `);
+    console.log("Sequence 'Profiles_id_seq' ownership altered.");
+    await queryInterface.sequelize.query(`
+      CREATE TABLE IF NOT EXISTS public."Profiles"
+      (
+        id integer NOT NULL DEFAULT nextval('"Profiles_id_seq"'::regclass),
+        "name" text COLLATE pg_catalog."default",
+        "accountSid" text COLLATE pg_catalog."default" NOT NULL,
+        "createdAt" timestamp with time zone NOT NULL,
+        "updatedAt" timestamp with time zone NOT NULL,
+        CONSTRAINT "Profiles_pkey" PRIMARY KEY ("id", "accountSid")
+      )
+    `);
+    console.log("Table 'Profiles' created.");
+    await queryInterface.sequelize.query(`
+      ALTER TABLE IF EXISTS public."Profiles" OWNER to hrm;
+    `);
+    console.log("Table 'Profiles' ownership altered.");
+
+    // Identifiers
+    await queryInterface.sequelize.query(`
+      CREATE SEQUENCE IF NOT EXISTS public."Identifiers_id_seq"
+        INCREMENT 1
+        START 1
+        MINVALUE 1
+        MAXVALUE 9223372036854775807
+        CACHE 1;
+    `);
+    console.log('Sequence "Identifiers_id_seq" created');
+    await queryInterface.sequelize.query(`
+    ALTER SEQUENCE public."Identifiers_id_seq" OWNER TO hrm
+    `);
+    console.log("Sequence 'Identifiers_id_seq' ownership altered.");
+    await queryInterface.sequelize.query(`
+      CREATE TABLE IF NOT EXISTS public."Identifiers"
+      (
+        id integer NOT NULL DEFAULT nextval('"Identifiers_id_seq"'::regclass),
```

Comment:
Do we want this table to permit duplicate identifiers for the same account or should we add a secondary constraint?